themes/godotengine/pages/privacy-policy.htm

```diff
@@ -29,9 +29,9 @@ <h3 class="title">The Information We Collect</h3>
     <li>you visit the Godot web site (<a href="/">godotengine.org</a>), hosted by <a href="https://www.tuxfamily.org/">TuxFamily</a> (see the TuxFamily privacy policy: <a href="https://faq.tuxfamily.org/Privacy/En">https://faq.tuxfamily.org/Privacy/En</a>);</li>
     <li>you access the Asset Library in the Godot editor application to browse or download assets;</li>
     <li>you sign up as a Godot Patreon (<a href="https://patreon.com/godotengine">https://patreon.com/godotengine</a>) or otherwise donate to Godot (<a href="/donate">https://godotengine.org/donate</a>);</li>
-    <li>you sign up and log in to an account on the Godot Chat platform (<a href="https://chat.godotengine.org" data-barba-prevent>https://chat.godotengine.org</a>), and use it to participate in private and public discussions;</li>
-    <li>you sign up and log in to an account on Godot's Q&amp;A site (<a href="https://godotengine.org/qa" data-barba-prevent>https://godotengine.org/qa</a>), and use it to ask or answer questions;</li>
-    <li>you sign up and log in to an account on Godot's Asset Library (<a href="https://godotengine.org/asset-library" data-barba-prevent>https://godotengine.org/asset-library</a>), and use it to submit content;</li>
+    <li>you sign up and log in to an account on the Godot Chat platform (<a href="https://chat.godotengine.org">https://chat.godotengine.org</a>), and use it to participate in private and public discussions;</li>
```

Comment:
`data-barba-prevent` should be unnecessary here as it's not the same subdomain. If I'm reading its source code right, Barba.js checks for the whole hostname to check whether the website is external, not just the domain name.